### PR TITLE
ci: add sync-docs to sync nextui docs

### PR DESCRIPTION
**Pull request by winchesHe**
## 📝 Description

When update docs in main branch 

And this ci will create a PR like to sync docs automatically https://github.com/nextui-org/nextui/pull/3079

## ✅ Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### CODE REVIEW

**Comment by jrgarciadev** [comment]
@winchesHe update this PR from main

@wingkwong please review it after updated

File: README.md [hunk 1]
```diff
@@ -84,22 +84,26 @@ NextUI CLI v0.2.0
...
 │
-◇  Next steps ───────────╮
-│                               │
-│  cd next-app-template         │
-│  npm install                  │
-│                               │
-├───────────────────╯
+◇  Next steps ───────────────────╮
+│                                │
+│  cd next-app-template          │
```
Comment (wingkwong) [inline README.md:103]: cd my-nextui-app

**Review by jrgarciadev — APPROVED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -148,6 +148,9 @@ nextui.hook('preAction', async (command) => {
   // Init cache
   initCache(noCache);
 
+  // Init cache
+  initCache();
```
Comment (jrgarciadev) [inline src/index.ts:152]: @winchesHe  is this correct?

File: .github/workflows/sync.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        uses: pnpm/action-setup@v2
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+        
+      - name: Set up Git
+        run: |
+          git config --global user.name 'winchesHe'
+          git config --global user.email '329487092@qq.com'
+
```
Comment (jrgarciadev) [inline .github/workflows/sync.yml:33]: @winchesHe can we put a generic github user?

Comment (winchesHe) [reply .github/workflows/sync.yml:33]: Sure, change it to github-bot

File: src/index.ts (hunk 2 above)
Comment (winchesHe) [reply src/index.ts:152]: i will fix, probally merge main error

**Review by jrgarciadev — APPROVED** [review]

**Pull request merged**